packages/kit/scripts/update_version.js

```diff
@@ -0,0 +1,16 @@
+import { readFile, writeFile } from 'node:fs/promises';
```

Comment:
I would add a TODO to remove this file when all versions of Node support import assertions in a non-experimental manner